suricata/update/loghandler.py

```diff
@@ -62,18 +64,58 @@ def emit(self, record):
             level_prefix = self.YELLOW
             message_prefix = ""
 
-        self.stream.write("%s%s%s - <%s%s%s> -- %s%s%s\n" % (
-            self.GREEN,
-            self.formatTime(record),
-            self.RESET,
-            level_prefix,
-            record.levelname.title(),
-            self.RESET,
-            message_prefix,
-            self.mask_secrets(record.getMessage()),
-            self.RESET))
+        if os.isatty(sys.stdout.fileno()) and \
```

Comment:
You can probably do something like `os.isatty(self.stream.fileno())` to get rid of the different cases for stderr and stdout. This should simply this method by not having to carry about logging level or output filestream.